### DISCUSSION TITLE
关于和 SnipDo联动

### DISCUSSION THREAD

**Discussion by liuyu6** [pot-app/pot-desktop#313] [Ideas]
作者你好，和 SnipDo联动有个新思路，目前 我在SnipDo 通过创建 快捷键文本扩展(快捷键就是唤醒软件的快捷键) 划词的时候唤醒翻译软件，程序响应速度也很快； 但也想学习下 执行shell脚本唤醒Pot的方法

**Comment by Pylogmon** [comment]
没看懂你的意思
